tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py

```diff
@@ -8,7 +8,7 @@
 
 class Browser(Base):
 
-    _browser_app_locator = (By.CSS_SELECTOR, 'div.browser[transition-state="opened"]')
+    _browser_app_locator = (By.CSS_SELECTOR, 'div[data-manifest-name="Browser"][transition-state="opened"]')
```

Comment:
This locator is already defined at `gaiatest/apps/search/app.py`. That make me wonder if we shouldn't make `Browser` inherit from `Search`. What do you think @mwargers ?